python/src/lib.rs

```diff
@@ -15,18 +15,107 @@
 // specific language governing permissions and limitations
 // under the License.
 
+use ballista::prelude::*;
+use datafusion::execution::SessionStateBuilder;
+use datafusion::prelude::*;
+use datafusion_python::context::PySessionContext as DataFusionPythonSessionContext;
```

Comment:
is there need to rename `PySessionContext` ?

Reply:
Not anymore, will change back, it was leftover from where the names was PySessionContext.